core/src/main/java/org/apache/iceberg/ManifestReader.java

```diff
@@ -226,11 +226,11 @@ private void cacheChanges() {
 
   @Override
   public Iterator<DataFile> iterator() {
-    return iterator(alwaysTrue(), fileSchema);
+    return iterator(fileSchema);
   }
 
   // visible for use by PartialManifest
-  Iterator<DataFile> iterator(Expression partFilter, Schema fileProjection) {
+  Iterator<DataFile> iterator(Schema fileProjection) {
```

Comment:
The purpose of this was to allow pushing filters into the manifest files if we ever use a format other than Parquet. I think overall this is a good change because we can add it back later. Just wanted to note why the unused argument was there.